package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mongoose-tsgen",
   "description": "A Typescript interface generator for Mongoose that works out of the box.",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

Comment:
Shouldn't this have been 6.0 since this is a breaking change?

Reply:
Yes you're totally right, just fixed!